src/fields/fe_value_handler.cc

```diff
@@ -114,7 +114,7 @@ void FEValueHandler<elemdim, spacedim, Value>::value_list(const std::vector< Poi
 	ASSERT_EQ( point_list.size(), value_list.size() ).error();
 
     DOFHandlerBase::CellIterator cell = dh_->mesh()->element( elm.idx() );
-	dh_->get_loc_dof_indices(cell, dof_indices);
+	dh_->get_dof_indices(cell, dof_indices);
```

Comment:
Check and change other calls to get_loc_dof_indices (in methods: interpolate, calculate_native_values, fill_data_to_cache), comment this method since it is currently wrong (the shift by loc_offset_ is not enough for local <-> global mapping. Only local to global mapping should be used in future. 

Check if we can merge methods where this method is used, in particular  calculate_native_values should use FeValueHandler.